sda-commons-server-dropwizard/src/main/java/org/sdase/commons/server/dropwizard/bundles/scanner/MappableField.java

```diff
@@ -0,0 +1,117 @@
+package org.sdase.commons.server.dropwizard.bundles.scanner;
+
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class MappableField {
```

Comment:
I wonder if it would make sense and be possible to make this class package private as it's only an implementation detail. We would need to move around some classes, I guess.

Reply:
It is the return type of the `JacksonTypeScanner`'s public `scan(…)` method. I put both in a separate package, because they both work together quite independent of the implementation that applies the configuration somewhere. I was also thinking about removing any Dropwizard relation and add the [specific types](https://github.com/SDA-SE/sda-dropwizard-commons/pull/2685/files#diff-c77741edf0b134673367e86c14cd853540daaf3ea5fd8210366d6c416c507dd2R23-R24) in the constructor.

Reply:
Just saw, that I could make the constructor package private. That should clarify enough that it is used internally.